### ISSUE TITLE
Only seems to support x64

### ISSUE THREAD

**Issue by evilC** [WootingKb/wooting-analog-wrappers#2]
I cannot seem to get the .NET SDK to work with AnyCPU or x86  
`wooting_analog_wrapper.dll` seems to be x64 only  
Any help?

**Comment by simon-wh** [comment]
That's because (at least at the moment) we're only building the analog SDK for 64-bit. So as a result AnyCPU and x86 won't work. Do you have a particular need for 32-bit builds? As we didn't set it up for them as we couldn't see a massive need, seeing as 64-bit CPUs have been mainstream for a pretty long time

**Comment by evilC** [comment]
I was looking at porting [WootingAHK](https://github.com/evilC/WootingAHK) over to the new API, and AutoHotkey users tend to install the x86 version, even on x64 machines

**Comment by simon-wh** [comment]
I looked into the 32/64-bit situation for AHK. By default with the installer it selects x64 and it appears that 64 bit is the direction that they're pushing into? Is the community still pretty strong on primarily using 32-bit?

We're trying to determine how needed 32-bit is, as to support it we'll probably have to make the system SDK DLL and all plugins 32-bit only so they can be loaded by both

**Comment by simon-wh** [comment]
From looking into it more, I've concluded that to properly support 32-bit, it would require 64-bit/32-bit versions of the wrapper, the full SDK bundled with Wootility and every first party and third party device plugin in the future. Although it is possible to do, requiring all current and future device plugins written by us and (hopefully) third parties in the future to support 32-bit and 64-bit at all times doesn't really make sense. As yes, 32-bit appears to still be alive in AHK, but it's not the default installation and appears to be legacy to facilitate old scripts. So given that and the scope of what's required for 32-bit support, I've decided that at least right now, we're not going to be pursuing 32-bit builds.
